integrated_channels/integrated_channel/learner_data.py

```diff
@@ -25,26 +25,30 @@ class BaseLearnerExporter(object):
     Base class for exporting learner completion data to integrated channels.
     """
 
+    GRADE_PASSING = 'Pass'
+    GRADE_FAILING = 'Fail'
+    GRADE_INCOMPLETE = 'In Progress'
+
     @property
     def grade_passing(self):
         """
         Returns the string used for a passing grade.
         """
-        return 'Pass'
+        return self.GRADE_PASSING
```

Comment:
It's not entirely clear to me what we get out of making this a property referring statically to the value stored on the class, rather than just storing the value on the class (or, as mentioned above, the subclass) with this name in the first place. Can you clarify what the goal is?

In conjunction with the above, I might actually keep these methods and make them raise `NotImplementedError`, thus forcing subclasses to override with the relevant values for that particular `IntegratedChannel`.